### PR TITLE
remove Rusanov penalties from the linear model

### PR DESCRIPTION
**Pull request by jkozdon**
Previously the linear model penalized discontinuities for all variables in the state vector, not just those being used in the linear model.

### CODE REVIEW

**Comment by jkozdon** [comment]
bors try

**Comment by jkozdon** [comment]
bors try

**Comment by jkozdon** [comment]
bors try

File: src/Atmos/Model/linear.jl [hunk 1]
```diff
@@ -1,4 +1,5 @@
 using CLIMAParameters.Planet: R_d, cv_d, T_0, e_int_v0, e_int_i0
+using ..DGmethods.NumericalFluxes: RusanovNumericalFlux
```
Comment (mwarusz) [inline src/Atmos/Model/linear.jl:3]:
```suggestion
import ..DGmethods.NumericalFluxes: update_penalty!
```

Comment (lcw) [reply src/Atmos/Model/linear.jl:3]: Thanks! You beat me too this.

**Comment by jkozdon** [comment]
This probably doesn't not sense to use / try until #1037 is merged.

File: src/Atmos/Model/linear.jl (hunk 1 above)
Comment (jkozdon) [reply src/Atmos/Model/linear.jl]: Thanks!

**Pull request closed without merging**